unix/README.md

```diff
@@ -1,161 +1,106 @@
 # Machine Stats for Unix-like systems
 
-Machine Stats for Linux/Unix leverages [Ansible](https://www.ansible.com/) to gather facts in a cross-platform way.
+[![PyPI](https://img.shields.io/pypi/v/machine-stats)](https://pypi.org/project/machine-stats/)
 
-## Data captured
-
-For *nix based systems, by default, the following metrics are captured from the resources and sent and stored in Tidal Migrations:
-
-```
-Host Name
-FQDN
-IP Addresses
-RAM Allocated (GB)
-RAM Used (GB)
-Storage Allocated (GB)
-Storage Used (GB)
-CPU Count
-Operating System
-Operating System Version
-CPU name
-```
-
-## Requirements
-
-1. **Git**
-
-    ```
-    # Debian/Ubuntu
-    sudo apt-get update
-    sudo apt-get install git
-
-    # Fedora
-    sudo dnf -y update
-    sudo dnf -y install git
-
-    # RHEL/CentOS
-    sudo yum update
-    sudo yum install git
-
-    # macOS (Xcode)
-    xcode-select --install
-
-    # macOS (brew)
-    brew install git
-    ```
+A simple and effective way to gather machine statistics (RAM, Storage, CPU)
+from a server environment as a first layer of a [Tidal Migrations discovery
+process](https://guides.tidalmg.com).
 
-    Or follow the installation instructions on [Git's web
-    site](https://git-scm.com/downloads)
-
-2. **`pipenv`**
-
-    ```
-    # Debian 10+/Ubuntu 20.04+
-    sudo apt-get update
-    sudo apt-get install pipenv
-
-    # Fedora 32+
-    sudo dnf -y update
-    sudo dnf -y install pipenv
-
-    # RHEL 7
-    sudo yum install rh-python38
-    scl enable rh-python38 bash
-    python3.8 -m pip install --user pipenv
-
-    # RHEL 8
-    sudo yum install python3
-    python3 -m pip install --user pipenv
-    ```
-
-    Or follow the installation instructions on [`pipenv` web
-    site](https://pipenv.pypa.io/en/latest/install/#installing-pipenv)
+Machine Stats for Linux/Unix leverages [Ansible](https://www.ansible.com/) to
+gather facts in a cross-platform way.
 
 ## Installation
 
-We are going to install Machine Stats for Unix-like systems in user's `$HOME`
-directory instead of system-wide, so the installation won't affect system
-libraries and packages.
-
-First of all, let's obtain the sources of Machine Stats.
-
-Open your terminal emulator and change the current working directory to your
-`$HOME` directory by running this simple command:
-
-```
-cd 
-```
-
-Let's fetch the sources from our Git repository:
+Install locally in a Python 3 environment:
 
 ```
-git clone https://github.com/tidalmigrations/machine_stats.git
+pip install machine-stats
 ```
 
-All of the code should be downloaded into a `machine_stats` directory, let's
-change into that directory:
+If you are using `pipx` you can install Machine Stats as the following:
 
 ```
-cd machine_stats
+pipx install machine-stats
 ```
 
-This directory contains scripts for both Window version and the version for
-Unix-like systems. We need the latter, so let's switch to the appropriate
-directory:
+or even run Machine Stats directly with
```

Comment:
I'm against this change, I don't want to provide additional support for `pipx`.

`pip install` is the recommended installation process. If the user is familiar with `pipx`, I think she knows what to do!

Reply:
Yea I agree, one recommended install approach is simpler. In the times I've worked with people using this, none are overly familiar with python or are python developers. With multiple methods here, it is confusing, even to me with some understanding of what the differences are, about what I _should_ do and why. Maybe it is best to then simply remove this entire example command from the Installation steps, to avoid confusion, as its not on the 'golden path'.

Reply:
Yes, agree! If it's possible to install with `pip install`, then it's possible to install/run with `pipx install`/`pipx run`.